adv-rumors.lua

```diff
@@ -8,76 +8,221 @@ Improves the "Bring up specific incident or rumor" menu in Adventure mode.
 
 - Moves entries into one line
 - Adds a "slew" keyword for filtering, making it easy to find your kills and not your companions'
+- Adds "gaining", "placement" and "giving" keywords for artifacts
 - Trims repetitive words
 
 ]====]
 
 --========================
--- Author : 1337G4mer on bay12 and reddit
--- Version : 0.2
--- Description : A small utility based on dfhack to improve the rumor UI in adventure mode.
+-- Author : 1337G4mer on bay12 and reddit, version 0.3 by Crystalwarrior on Discord
+-- Version : 0.3
+-- Description : An utility based on dfhack to improve the rumor, directions and ask about... UIs in adventure mode.
```

Comment:
```suggestion
-- Description : A utility based on dfhack to improve the rumor, directions and "ask about..." UIs in adventure mode.
```
not sure if the "based on dfhack" is necessary any more, since it's true for everything in this repo by definition